src/devices/Bmp180/nuget.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This file isn't required since we already have a global one that includes these two package sources.